Makefile.in

```diff
@@ -15,7 +15,7 @@ export OCAMLFIND_INSTFLAGS=-patch-version "$(VERSION)"
 
 build: all opt
 all: byte-code-library 
-opt: native-code-library
+opt: native-code-library mysql.cmxs
```

Comment:
afaik `ocamllopt -shared` is not available everywhere, so need to do a configure check and enable this target conditionally